keyboards/handwired/ortho5x14/ortho5x14.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2021 Richard Nunez
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+  k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, \
+  k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, \
+  k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, \
+  k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, \
+  k40, k41, k42, k43, k44, k45,           k48, k49, k4a, k4b, k4c, k4d  \
+) \
+{ \
+  { k00, k01, k02, k03, k04, k05,  k06,    k07, k08, k09, k0a, k0b, k0c, k0d }, \
+  { k10, k11, k12, k13, k14, k15,  k16,    k17, k18, k19, k1a, k1b, k1c, k1d }, \
+  { k20, k21, k22, k23, k24, k25,  k26,    k27, k28, k29, k2a, k2b, k2c, k2d }, \
+  { k30, k31, k32, k33, k34, k35,  k36,    k37, k38, k39, k3a, k3b, k3c, k3d }, \
+  { k40, k41, k42, k43, k44, k45, KC_NO, KC_NO, k48, k49, k4a, k4b, k4c, k4d } \
+}
+
+#define COMPACT_KEYMAP( \
+  k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d,\
+  k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d,\
+  k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d,\
+  k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d,\
+  k40, k41, k42, k43, k44,    k45,       k46,  k47, k48, k49, k4a, k4b \
+) \
+{ \
+  {      k00, KC_##k01, KC_##k02, KC_##k03,      k04, KC_##k05, KC_##k06, KC_##k07, KC_##k08, KC_##k09, KC_##k0a, KC_##k0b,      k0c,      k0d }, \
+  { KC_##k10,      k11, KC_##k12, KC_##k13, KC_##k14, KC_##k15, KC_##k16, KC_##k17, KC_##k18, KC_##k19, KC_##k1a, KC_##k1b,      k1c,      k1d }, \
+  { KC_##k20, KC_##k21, KC_##k22, KC_##k23, KC_##k24, KC_##k25, KC_##k26, KC_##k27, KC_##k28, KC_##k29, KC_##k2a,      k2b,      k2c, KC_##k2d }, \
+  { KC_##k30,      k31, KC_##k32, KC_##k33, KC_##k34, KC_##k35, KC_##k36, KC_##k37, KC_##k38,      k39,      k3a,      k3b,      k3c, KC_##k3d }, \
+  { KC_##k40, KC_##k41, KC_##k42, KC_##k43, KC_##k44, KC_##k45, KC_NO,    KC_NO,    KC_##k46, KC_##k47, KC_##k48, KC_##k49, KC_##k4a, k4b } \
+}
```

Comment:
We actively discourage this type of layout macro, as all it really does is cause confusion for newer users, and isn't supported by the configurator. 
```suggestion
```

Reply:
Done.